solr/conf/schema.xml

```diff
@@ -143,10 +143,8 @@
     <fieldType name="location" class="solr.LatLonType" subFieldSuffix="_d"/>
 
     <fieldType name="location_rpt" class="solr.SpatialRecursivePrefixTreeFieldType"
-               distErrPct="0.025"
-               maxDistErr="0.000009"
-               units="degrees"
-            />
+               geo="true" distErrPct="0.025" maxDistErr="0.001" distanceUnits="kilometers"/>
+
```

Comment:
Why revert back? If the schema is versioned alongside the code.. I'm ok with it. `degrees` is more relevant to non-geo spatial distances, I think `kilometers` is more consistent with geo ones? Discuss on slack?